docs/guides/src/operation/black_list.md

```diff
@@ -0,0 +1,31 @@
+# Black list
+
+## Read black list
+If you want to reject to query to a table, you can add table to 'read_reject_list'.
+
+### Example
+```
+curl --location --request POST 'http://localhost:5000/reject' \
+--header 'Content-Type: application/json' \
+-d '{
+    "operation":"Add",
```

Comment:
There are three operations in total that you can list.

Reply:
I added set and remove operations